finalPrep/modZeroHero.js

```diff
@@ -2,51 +2,74 @@
 
 // Declare two variables - heroName AND specialAbility - set to strings
 
+var heroName = "Rock 'Em Sock 'Em"
+var specialAbility = "super punch"
+
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
 
+var greeting = ("Hey y'all, I'm" + heroName)
+var catchphrase = (`here to ${specialAbility} and take names`)
+
 // Declare two variables - power AND energy - set to integers
 
+var power = 12
+var energy = 8
+
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
 
-// Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var fullPower = (power * 500)
+var fullEnergy = (energy + 150)
 
+// Declare two variables - isHuman and identityConcealed - assigned to booleans
+var isHuman = true
+var identityConcealed = false
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
-
+var archEnemies = ["Madam Evil", "Dr. Mess", "Lady Lazarus"]
+var sidekicks = ["Lil' Slugger", "Big Bad Boi", "Roger Roger"]
 // Print the first sidekick to your console
-
+console.log(sidekicks[0]);
 // Print the last archEnemy to the console
-
+console.log(archEnemies[2]);
 // Write some code to add a new archEnemy to the archEnemies array
-
+archEnemies.push("Lester Loco");
 // Print the archEnemies array to console to ensure you added a new archEnemy
-
+console.log(archEnemies);
 // Remove the first sidekick from the sidekicks array
-
+archEnemies.pop(0);
 // Print the sidekicks array to console to ensure you added a new sidekick
-
+console.log(sidekicks);
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
+function assessSituation (dangerLevel, saveTheDay, badExcuse){
+  if (dangerLevel < 10) {
+    console.log("Meh. Hard pass.");
+  } else if (dangerLevel <= 50){
+    console.log(saveTheDay);
+  } else {
+    console.log(badExcuse);
+  }
+}
```

Comment:
If the `dangerLevel == 10`, shouldn't we have the badExcuse printed to the console? Did you try running your code to see if this would happen?